### DISCUSSION TITLE
Just wanna know when will the V2 be available

### DISCUSSION THREAD

**Discussion by Mrw33554432** [acheong08/ChatGPT#882] [General]
I don't quite know if the project dev is a prime member; if he is, then what he says is true that open ai breaks it. But if he is not, then it's possible that the portal is only available for paid members. So I just wanna check on that, and if he needs it, I will do my best to help him.

**Comment by acheong08** [comment]
I have no Plus account but people with plus accounts have tested it and confirmed that it doesn't work
